app/views/layouts/application.html.erb

```diff
@@ -4,6 +4,9 @@
     <title><%= "#{@project.name} - " if @project %><%= t('cm-central') %></title>
     <%= csrf_meta_tag %>
     <%= stylesheet_link_tag :application %>
+  <%=javascript_include_tag "https://www.authy.com/form.authy.min.js" %>
+  <%=stylesheet_link_tag "https://www.authy.com/form.authy.min.css" %>
```

Comment:
small style problem here: add one more level of indentation and a space after `<%`